lib/components/Loaders/BlockLoader/BlockLoader.tsx

```diff
@@ -0,0 +1,26 @@
+interface BlockLoaderProps {
+  height: number
+  cols: number | string
+}
+
+interface IGridColsClass {
+  [key: string | number]: string
+}
+
+const gridColsClass: IGridColsClass = {
```

Comment:
Is there a way we can leverage the grid styles that live in /utils/optionClasses.tsx as these are in a const named rdsGridColumns

Reply:
I have fixed this. Going to merge it.